data/korath/korath jobs.txt

```diff
@@ -243,3 +243,593 @@ mission "Kor Efreti to Wanderers: Farming"
 	on complete
 		payment 10000 200
 		dialog `When you return, the Korath move their hands in a circular motion with their palms toward you as a sign of gratitude. They hand you <payment> and disembark your ship.`
+
+# This is your key to entering La'ab Bamen without a bribe.
+# Hence the lack of randomness.
+mission "Korath Humanika: Fish Trade (small)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 5 10 0.8
+	illegal 50000
```

Comment:
These are a unique species of fish native to New Tortuga, which require precise conditions that exist almost nowhere else. The only authorities in the area (Syndicate and Republic) would know about the trade. They certainly would fine pilots involved in it.

Other authorities wouldn't know about it, but I'd like to put off fixing *that* problem until we have a way to make illegality government-dependent. That way, we can fix it the same way everywhere, instead of removing whatever bizarre workaround I would put in here.

Reply:
The problem is not the fish, nor the choice of trade partners, but rather the items being sent back to New Tortuga. You're supposed to be left with the impression that the trade can involve more dangerous payment than waste processing systems. I didn't want to say that outright in the text, since I wanted the player to suspect or infer it.

Reply:
I added a comment to the txt. How does this sound?

> `   # The mission is illegal because of the broader trade agreement. The player is supposed to be left with the impression that the Korath are sending back more dangerous things than waste processing systems. The Syndicate (and by extension, the Republic) is aware of this, and fines pilots involved.`
> `   illegal 50000`

Reply:
Having the fish indicates you're already involved, though. It is common for groups to sanction ordinary goods that are being shipped to groups they disagree with. The US and Europe do a lot of that.

I believe the term is "targeted sanctions."

In this situation, it seems like a reasonable approach for the Republic to take, and well within their personality. They don't want to punish every day New Tortugans, who aren't even aware of the arrangement. So, they punish people directly involved in the dangerous trade arrangement.

Reply:
No, the cargo scanners are able to pick out all sorts of arbitrarily-specific things in missions and jobs.

How do the scanners tell the difference between endangered animals and very similar species that are not endangered?
How do they know my knock-off paintings aren't real?
What part of my silver spoons do they scan to determine they're stolen?
How did they know I never paid taxes on my "untaxed wine?"
Why do they think my thumbscrews are torture devices, not museum pieces?
What advanced scanning technology lets them determine the IDs in my cargo hold are forged?
Are they truly able to walk the thin grey line between cybernetic chickens and weaponized chickens?

The answer: the cargo scanners detect whatever they need to detect to make the story happen.

Reply:
Yea, this isn't any more suspension than other current arrangements, and being particular species of fish makes it significantly more plausible than many others.

And no individual pilot needs to know -why- that's on the list of contraband they enforce, it's on the list, registered and verified by the computer, and that's plenty good enough for citations and fines.

Reply:
You can now explicitly ask why the mission is illegal. Jenny will give you what information she has, and her interpretation.